lib/livebook/application.ex

```diff
@@ -125,19 +125,28 @@ defmodule Livebook.Application do
     :persistent_term.put(:livebook_local_file_system, local_file_system)
   end
 
-  defp validate_epmd_module!() do
+  defp set_epmd_module!() do
     # We use a custom EPMD module. In releases and Escript, we make
     # sure the necessary erl flags are set. When running from source,
-    # those need to be passed explicitly.
+    # we try to use the new :kernel configuration available in OTP 27.2,
+    # otherwise it needs to be set explicitly.
+
+    # TODO: always rely on :kernel configuration once we require OTP 27.2
+
     case :init.get_argument(:epmd_module) do
       {:ok, [[~c"Elixir.Livebook.EPMD"]]} ->
         :ok
 
       _ ->
-        Livebook.Config.abort!("""
-        You must set the environment variable ELIXIR_ERL_OPTIONS="-epmd_module Elixir.Livebook.EPMD" \
-        before the command (and exclusively before the command)
-        """)
+        Application.put_env(:kernel, :epmd_module, Livebook.EPMD, persistent: true)
+
+        # Note: this is a private API
+        if :net_kernel.epmd_module() != Livebook.EPMD do
```

Comment:
Ideally we could check if we are on OTP 27.2, but version minor is not accessible, right?

Reply:
Yeah. you could get the version of the kernel or stdlib applications though. I think this is fine as is.